analysis_engine/features.py

```
@@ -0,0 +1,271 @@
+        are combined for robustness.
+        """
+        nose_x = float(pixels[lms.NOSE][0])
+        left_x = float(pixels[lms.LEFT_ANKLE][0])
+        right_x = float(pixels[lms.RIGHT_ANKLE][0])
+        head_vote = "left" if abs(nose_x - left_x) < abs(nose_x - right_x) else "right"
+
+        left_toe_out = float(pixels[lms.LEFT_FOOT_INDEX][0] - pixels[lms.LEFT_HEEL][0])
+        right_toe_out = float(pixels[lms.RIGHT_FOOT_INDEX][0] - pixels[lms.RIGHT_HEEL][0])
+        toe_vote = "left" if abs(left_toe_out) > abs(right_toe_out) else "right"
+
+        front = head_vote if head_vote == toe_vote else head_vote
```

Ruff (0.16.1)

[warning] Useless `if`-`else` condition

(RUF034)